docs/content/en/docs/administration/permissions/list_available_assignees.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "list_available_assignees"
+linkTitle: "list_available_assignees"
+weight: 17
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``list_available_assignees(workspace_id: str, dashboard_id: str) -> CatalogAvailableAssignees``
+
+Provide managing dashboard permissions for user and user groups.
```

Comment:
This one lists users and user groups that can be assigned dashboard permissions for the dashboard in the workspace.